test/disable-goosig-test.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * test/disable-goosig-test.js - Test disabling GooSig
+ * Copyright (c) 2020, The Handshake Developers (MIT License).
+ * https://github.com/handshake-org/hsd
+ */
+
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('bsert');
+const fs = require('fs');
+const {resolve} = require('path');
+const FullNode = require('../lib/node/fullnode');
+const AirdropProof = require('../lib/primitives/airdropproof');
+const BlockTemplate = require('../lib/mining/template');
+const consensus = require('../lib/protocol/consensus');
+
+// Use a valid, working airdrop proof
+const AIRDROP_PROOF_FILE = resolve(__dirname, 'data', 'airdrop-proof.base64');
+const rawAirdropProof =
+  Buffer.from(fs.readFileSync(AIRDROP_PROOF_FILE, 'binary'), 'base64');
+const proof = AirdropProof.decode(rawAirdropProof);
+
+let node, networkAirdrop;
```

Comment:
nit: because we intend to create a new `node` every internal `describe` block, perhaps we should declare a new node for each `describe` block. Declaring the variable globally implies that we will be using the same instance across the tests, but this PR does a great job of isolating test scenarios.